transport/tchannel/channel_transport.go

```diff
@@ -82,11 +82,12 @@ func (options transportOptions) newChannelTransport() *ChannelTransport {
 		logger = zap.NewNop()
 	}
 	return &ChannelTransport{
-		once:   lifecycle.NewOnce(),
-		ch:     options.ch,
-		addr:   options.addr,
-		tracer: options.tracer,
-		logger: logger,
+		once:            lifecycle.NewOnce(),
+		ch:              options.ch,
+		addr:            options.addr,
+		tracer:          options.tracer,
+		logger:          logger,
+		exactCaseHeader: options.exactCaseHeader,
```

Comment:
should we call it `ExactHeaderItems` or something to match the name on the `Headers`?